presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -235,6 +239,20 @@ public HiveClientConfig setForceLocalScheduling(boolean forceLocalScheduling)
         return this;
     }
 
+    public NodeSelectionStrategy getNodeSelectionStrategy()
+    {
+        return nodeSelectionStrategy;
+    }
+
+    @Config("hive.node-selection-strategy")
+    public HiveClientConfig setNodeSelectionStrategy(String nodeSelectionStrategy)
```

Comment:
nit: `NodeSelectionStrategy nodeSelectionStrategy`, and remove the `String -> Enum` handling below.